src/main/java/com/uid2/admin/auth/OktaAuthProvider.java

```diff
@@ -11,16 +12,34 @@
 import com.okta.jwt.JwtVerifiers;
 import static com.uid2.admin.auth.AuthUtil.isAuthDisabled;
 import java.time.Duration;
-import java.util.Arrays;
 import java.util.List;
-import java.util.Collections;
 
 public class OktaAuthProvider implements AuthProvider {
     private final JsonObject config;
-    private List<String> scopes = Collections.unmodifiableList(Arrays.asList("openid",  "profile" ,"email" ,"groups"));
-
+    private final List<String> scopes = List.of("openid", "profile", "email", "uid2.admin.human", "offline_access");
```

Comment:
do we still need these scopes ? I don't see them referenced anywhere

Reply:
just openid, email, and uid2.admin.human (openid is required by default, email to display the user logged in in the UI)